ui/src/utils/pluginUtils.ts

```diff
@@ -0,0 +1,23 @@
+const KNOWN_PUBLIC_PLUGINS = new Set([
```

Comment:
Is there a way to fetch this list from an endpoint somehow? As we do develop a lot of plugins per month, I'd hate the idea to have to update the list manually with each new one added.